app/templates/views/add-service.html

```diff
@@ -11,8 +11,8 @@
 
 {% block maincolumn_content %}
 
-<div class="grid-row">
-  <div class="column-two-thirds">
+<div class="govuk-govuk-grid-row">
```

Comment:
Too many many `govuk`s here
```suggestion
<div class="govuk-grid-row">
```